apps/smart-contracts/core/test/unit/PrePOMarket.test.ts

```diff
@@ -705,20 +714,35 @@ describe('=> prePOMarket', () => {
       const shortToRedeem = amountMinted
       const totalOwed = await calculateTotalOwed(longToRedeem, shortToRedeem, false)
 
-      await prePOMarket.connect(user).redeem(longToRedeem, shortToRedeem)
+      await prePOMarket.connect(user).redeem(longToRedeem, shortToRedeem, user.address)
 
       expect(await longToken.balanceOf(user.address)).to.eq(amountMinted.sub(longToRedeem))
       expect(await shortToken.balanceOf(user.address)).to.eq(amountMinted.sub(shortToRedeem))
       expect(await collateralToken.balanceOf(user.address)).to.eq(totalOwed)
     })
 
+    it('correctly settles redemption with recipient != redeemer before market end', async () => {
```

Comment:
nit: `correctly` is implied, since it's not a revert test case, and should follow the `[result] if [conditions]` test naming convention
```suggestion
    it('redeems if funder != recipient and market not ended', async () => {
```